### ISSUE TITLE
Error: Cannot find module 'mout/lang'

### ISSUE THREAD

**Issue by rbatllet** [bower/bower#2545]
Please, change the **mout** version dependency in package.json:

```
  "dependencies": {
    ...
    "mout": "^1.0.0",
    ...
  }
```

The new  **mout 1.2.0** is incompatible with this module, but not **mout 1.1.0**

The dependency change I propose is:

```
  "dependencies": {
    ...
    "mout": "~1.1.0",
    ...
  }
```
This solves the problem:

```
Error: Cannot find module 'mout/lang'
Require stack:
- /Users/user/myproject/node_modules/bower-config/lib/Config.js
```

**Comment by dariusve** [comment]
I have the same issue, I made the change to the bower-config package.json, but  I cannot make it work.

**Comment by rbatllet** [comment]
Add this in your package.json:

```
"dependencies": {
    ...
    "mout": "~1.1.0",
    ...
  }

```
Remove the **package-lock.json** and the **node_modules** directory. Run "**npm install**" 

It solves the problem.

**Comment by dariusve** [comment]
Thanks, it worked!

**Comment by rbatllet** [comment]
Finally it's solved in mout 1.2.2.
